spec/spec_helper.rb

```diff
@@ -102,22 +102,4 @@
   #   # test failures related to randomization by passing the same `--seed` value
   #   # as the one that triggered the failure.
   #   Kernel.srand config.seed
-  def stub_omiauth
-    OmniAuth.config.test_mode = true
-    OmniAuth.config.mock_auth[:google] =
-      OmniAuth::AuthHash.new(
-        {
-          provider: 'google',
-          uid: '123545',
-          info: {
-            name: 'John Doe',
-            email: 'johndoe@doe.com',
-            location: 'Doe World',
-            image: 'image_url'
-          },
-          extra: { raw_info: { hd: '@tamu.edu' } }
-        }
-      )
-    # etc.
-  end
```

Comment:
Is there a reason that this was removed?

Reply:
moved it into the spec test instead because of problems with calling the function from the helper